index.html

```diff
@@ -0,0 +1,499 @@
+        </div>
+      </div>
+      <div class="email-grid newsletter flex-col-aic">
+        <div class="news-griditem1">
+          <div class="news-picture fifth-color"></div>
+        </div>
+        <div class="news-griditem">
+          <article>
+            <div class="content"></div>
+          </article>
+        </div>
+        <div class="news-griditem">
+          <article>
+            <div class="content"></div>
+          </article>
+        </div>
+        <div class="news-griditem">
+          <article>
+            <div class="content"></div>
+          </article>
+        </div>
+      </div>
+    </div>
+
+    <!-- footer section starts here -->
+    <div class="footer flex-col-aic fourth-color">
+      <div class="feature-name browse color-black"></div>
+      <div class="tag-link flex-col-aic">
+        <div class="top-tag flex-row">
+          <span class="tag-span color-black"></span>
+          <span class="tag-span color-black"></span>
+          <span class="tag-span color-black"></span>
+          <span class="tag-span color-black"></span>
+          <span class="tag-span color-black"></span>
+          <span class="tag-span color-black"></span>
+          <span class="tag-span color-black"></span>
+        </div>
+        <div class="top-tag flex-row">
+          <span class="tag-span color-black"></span>
+          <span class="tag-span color-black"></span>
+          <span class="tag-span color-black"></span>
+          <span class="tag-span color-black"></span>
+          <span class="tag-span color-black"></span>
+          <span class="tag-span color-black"></span>
+          <span class="tag-span color-black"></span>
+        </div>
+        <div class="top-tag flex-row">
+          <span class="tag-span color-black"></span>
+          <span class="tag-span color-black"></span>
+          <span class="tag-span color-black"></span>
+          <span class="tag-span color-black"></span>
+        </div>
+      </div>
+      <div class="foot-para-div">
+        <p class="foot-para color-black">one</p>
+        <p class="foot-para color-black">two</p>
+        <p class="foot-para color-black">three</p>
+        <p class="foot-para color-black">three</p>
+      </div>
+      <div class="foot-lower flex-row-jsa">
+        <div class="foot-box color-black"></div>
+        <div class="lower-tag-div flex-col-aic">
+          <div class="top-tag flex-row">
+            <span class="tag-lw-span color-black"></span>
+            <span class="tag-lw-span color-black"></span>
+            <span class="tag-lw-span color-black"></span>
+            <span class="tag-lw-span color-black"></span>
+          </div>
+          <div class="top-tag flex-row">
+            <span class="tag-lw-span color-black"></span>
+            <span class="tag-lw-span color-black"></span>
+            <span class="tag-lw-span color-black"></span>
+          </div>
+        </div>
+        <div class="foot-box color-black"></div>
+      </div>
```

Comment:
- [ ] The footer contains centered links, so please convert these span tags into the proper HTML tag to make them links.